stdlib/public/core/DebuggerSupport.swift

```diff
@@ -298,18 +225,15 @@ public enum _DebuggerSupport {
         parentCollectionStatus: collectionStatus,
         refsAlreadySeen: &refsAlreadySeen,
         maxItemCounter: &maxItemCounter,
-        targetStream: &targetStream)
+        target: &target)
       printedElements += 1
     }
   }
 
-  // LLDB uses this function in expressions, and if it is inlined the resulting
```

Comment:
Is this comment stale?

Reply:
Yep! I'm going to land this, and then will take care of that in a follow-up PR, at the same time as moving those global symbols.